staging/src/k8s.io/client-go/util/exec/exec.go

```diff
@@ -26,6 +26,14 @@ type ExitError interface {
 	ExitStatus() int
 }
 
+// NewCodeExitError returns a new CodeExitError.
+func NewCodeExitError(err error, code int) *CodeExitError {
+	return &CodeExitError{
+		Err:  err,
+		Code: code,
+	}
+}
```

Comment:
(optional) if you just inline this, you don't need to wait for `client-go/util` approval.

Reply:
I don't understand the purpose of this constructor? It is not doing anything that clients can't do for themselves?